doc/Roadmap.md

```diff
@@ -7,18 +7,19 @@ Please feel free to comment!
 * Push notifications:
     * [XEP-0357: Push Notifications](https://xmpp.org/extensions/xep-0357.html)
     * Amazon AWS SNS (Simple Notification Service)
-* Instant Stream Resumption
 * Full text search in archive
+* Experimental pipelining
+* Instant Stream Resumption
```

Comment:
But we can still try to deliver this, as soon as new `ISR` is accepted as draft which I see possible.